src/components/ScenarioComponents/CardsComponents/GroupFilter/FiltersContainer.tsx

```diff
@@ -6,16 +6,16 @@ import FilterButton from './FilterButton';
 import FilterCard from './FilterCard';
 import React from 'react';
 import {Localization} from 'types/localization';
-import {getScenarioPrimaryColor} from 'util/Theme';
-import {Dictionary} from 'util/util';
 
 interface FiltersContainerProps {
+  /** Id of the filter container */
+  id: string;
+
+  color: string;
```

Comment:
Comment missing on what the color prop is for.
```suggestion

  /** Color of the filter container (should match respective scenario) */
  color: string;
```